packages/react/src/component-tests/IcBreadcrumb/IcBreadcrumb.cy.tsx

```diff
@@ -56,6 +58,28 @@ describe("IcBreadcrumb end-to-end tests", () => {
     cy.get('[page-title="Coffee"]').should(NOT_BE_VISIBLE);
   });
 
+  it("should switch to the back arrow then back to the chevron", () => {
```

Comment:
Please could you alter the test title to be more descriptive, `should switch from default view to back arrow with parent breadcrumb when backBreadcrumbOnly prop is set on IcBreadcrumbGroup and revert when prop is removed`